packages/web/src/questionnaire/trackResults.ts

```diff
@@ -20,11 +19,80 @@ export const results = {
     static: true
   },
   config: {
-    showAlertResults: false,
-    showAlertNoResults: true,
+    noResultsMessage: {fr: "Aucune aide n'a pu être identifiée avec les critères choisis..."},
+    noResultsImage: 'images/tracks/no-results.svg',
     showResultsTitle: false,
     showProgramInfos: true,
-    showProgramSubtitles: false
+    showProgramSubtitles: false,
+    filters: [
+      {
+        field: "nature de l'aide",
+        label: "nature de l'aide",
+        trueIf: '==',
+        values: [
+          { 
+            label: 'Accompagnement',
+            value: 'accompagnement'
+          },
+          { 
+            label: 'Financement',
+            value: 'financement'
+          },
+          { 
+            label: 'Prêt',
+            value: 'prêt'
+          },
+          { 
+            label: 'Formation',
+            value: 'formation'
+          },
+          // for debugging purposes
+          // { 
+          //   label: 'FAIL',
+          //   value: 'xxx'
+          // }
+        ]
+      },
+      {
+        field: "publicodes.est dans les objectifs de l'entreprise.une de ces conditions",
```

Comment:
question: why the field name is long like this ? Can we have something more like: `priority_objectives`

the select looks like:
```html
<select class="fr-select" id="publicodes.est dans les objectifs de l'entreprise.une de ces conditions" name="publicodes.est dans les objectifs de l'entreprise.une de ces conditions">
...
</select>
```

Is not very significant for a `id` and `name` on input form

Reply:
note: the `trackResults.ts` file is a configuration file, so the `field` must indicate the correct path to find the value in a program's yaml file (it could be long w/ publicodes.

In another hand indeed the `<select>` id and name could be set w/ the `label` key instead of the `field` key to be simpler. I made that change in a new commit. I initially chose to set id and name w/ the `field` key because there was less chance to set these select w/ the same value twice, but this risk is indeed not that high